server/src/api/v1/Invite/index.ts

```diff
@@ -0,0 +1,85 @@
+import moment from 'moment';
+import Router from '../../../lib/router';
+import sendMail from '../../../lib/mail';
+import emailTemplate from '../../../lib/mail/emailTemplate';
+import logger from '../../../logger';
+import { Invite } from '../../../models';
+import { DEFAULT_INVITE_EXPIRE_TIME_HOURS } from '../../../models/Invite/index.model';
+import UserService from '../../../lib/user';
+
+const router = Router();
+
+router.post('/:inviteId/verify', async (req, res) => {
+  const { inviteId } = req.params;
+  let invite: Invite = null;
+  try {
+    invite = await Invite.findOne({
+      where: {
+        id: inviteId,
+      },
+    });
+  } catch (e) {
+    res.json({
```

Comment:
you can not be sure the error is invite not found here (connection issue for example)